Googler.py

```diff
@@ -37,9 +37,28 @@ def __init__(self, attributes):
         self.location = attributes.get("location")
         self.lowest_dir_id = attributes.get("lowest_dir_id")
         self.job_family = attributes.get("job_family")
-        self.resources = []
+        self.resources = [] #List of Resource objects from resource.py module
```

Comment:
In Kayla's dictionary file that you might use to load in it is a set of resources not a list.

Reply:
Noted. We'll talk about the file format and decide the best way to ingest the information.